starters/basic-starter/package.json

```diff
@@ -15,7 +15,7 @@
     "classnames": "^2.3.1",
     "drupal-jsonapi-params": "^2.1.0",
     "html-react-parser": "^1.4.8",
-    "next": "^12.2.0",
+    "next": "^12.2.3",
```

Comment:
Was there a particular reason for changing the version constraint in package.json? Usually we shouldn't have to increase the version constraint for patch releases

Reply:
No reason I just didn't know patch releases didn't need an update